src/Applications/LDAS_App/ldas_setup

```diff
@@ -113,6 +113,7 @@ class LDASsetup:
         self.ladas_coupling = 0
         self.in_rstfile = None
         self.in_tilefile = 'None' # default string
+        self.ens_id_width = 6  # _exxxx
```

Comment:
@weiyuan-jiang: I think it would be better to use ENS_ID_WIDTH=4 as the default, where 4 refers to the number of **numerical** digits in the ensid_string.  That is, add the extra 2 to the field with (for "__e") as needed in the scripts and source code.  Then existing LDAS.rc files would still work. 
(Of course, a better name for ENS_ID_WIDTH would be ENS_ID_DIGITS, but that would not be backward compatible for existing LDAS.rc files...)

Reply:
good point. I will change it to 4